test/metabase/query_processor_test/alternative_date_test.clj

```diff
@@ -219,11 +219,11 @@
                               (assoc (mt/mbql-query times)
                                      :middleware {:format-rows? false}))))))))
 
-      (testing "bigquery adds UTC"
+      (testing "bigquery"
         (mt/test-drivers #{:bigquery-cloud-sdk}
-          (is (= [[1 "foo" #t "2004-10-19T10:23:54Z[UTC]" #t "2004-10-19T00:00Z[UTC]" #t "10:23:54"]
-                  [2 "bar" #t "2008-10-19T10:23:54Z[UTC]" #t "2008-10-19T00:00Z[UTC]" #t "10:23:54"]
-                  [3 "baz" #t "2012-10-19T10:23:54Z[UTC]" #t "2012-10-19T00:00Z[UTC]" #t "10:23:54"]]
+          (is (= [[1 "foo" #t "2004-10-19T10:23:54" #t "2004-10-19" #t "10:23:54"]
+                  [2 "bar" #t "2008-10-19T10:23:54" #t "2008-10-19" #t "10:23:54"]
+                  [3 "baz" #t "2012-10-19T10:23:54" #t "2012-10-19" #t "10:23:54"]]
```

Comment:
This means this "bigquery" test can be deleted and you can add `:bigquery-cloud-sdk` to the list of drivers on line 181.

Reply:
good catch. updated it ee2179f55d6448b1153ef4b4b5ce7f73b0a6ec19